armory/__main__.py

```diff
@@ -721,7 +721,7 @@ def exec(command_args, prog, description):
         sys.exit(1)
 
     args = parser.parse_args(armory_args)
-    armory.logs.update_filters(args.log_level)
+    armory.logs.update_filters(args.log_level, args.debug)
```

Comment:
I am good with this here, if you checkout `update_armory` branch, I cleaned this up quite a bit so we can deal with that when we get there